### PR TITLE
translate dogepedia.md into Chinese

### PR DESCRIPTION
**Pull request by shuferhoo**
translate dogepedia.md into Chinese

### CODE REVIEW

File: content/zh-cn/dogepedia.md [hunk 1]
```diff
@@ -1,75 +1,75 @@
...
-- [How do I get a wallet?](/dogepedia/articles/how-do-i-get-a-wallet)
-- [How do I backup my wallet?](/dogepedia/articles/how-to-backup-a-wallet)
-- [What are the benefits of a hardware wallet, and how to use it with Doge?](/dogepedia/articles/dogecoin-hardware-wallets)
-- [How do I recover a lost wallet?](/dogepedia/articles/recover-a-lost-wallet)
-- [What is a wallet, really?](/dogepedia/articles/what-is-a-wallet)
+### 钱包 {#wallets}
+你的狗狗币之旅将从这里开始，拥有一个钱包你就可以开始储蓄或者花费狗狗币了。
+- [如何获得钱包？](/zh-cn/dogepedia/articles/how-do-i-get-a-wallet)
+- [如何备份钱包？](/zh-cn/dogepedia/articles/how-to-backup-a-wallet)
+- [硬件钱包的有点以及如何使用？](/zh-cn/dogepedia/articles/dogecoin-hardware-wallets)
```
Comment (bobdos) [inline content/zh-cn/dogepedia.md:25]: 硬件钱包的有点->硬件钱包的优点

File: content/zh-cn/dogepedia.md [hunk 2]
```diff
@@ -1,75 +1,75 @@
...
+- [如何找回丢失的钱包？](/zh-cn/dogepedia/articles/recover-a-lost-wallet)
+- [钱包到底是什么，是真实存在的吗？](/zh-cn/dogepedia/articles/what-is-a-wallet)
 
-### Using your Doge {#using-doge}
-Dogecoin is a cryptocurrency, and you can use it to send and receive payments.
-- [How do I send and receive Dogecoin?](/dogepedia/articles/send-and-receive-dogecoin)
-- [How can I use Dogecoin to buy stuff from my favorite stores?](/dogepedia/articles/using-dogecoin-in-a-store)
-- [Is my privacy protected while using Dogecoin?](/dogepedia/articles/dogecoin-and-privacy)
+### 狗狗币的使用 {#using-doge}
+狗狗币是一种货币，你可以用它来付款和收款。
```
Comment (bobdos) [inline content/zh-cn/dogepedia.md:30]: 狗狗币是一种货币->狗狗币是一种加密货币

**Review by bobdos — CHANGES_REQUESTED** [review]

**Comment by shuferhoo** [comment]
@bobdos

**Review by bobdos — APPROVED** [review]
all good and it's ready to go.

many thanks for your contribution.

cheers.

**Pull request merged**